lib/spaceapi.py

```diff
@@ -11,10 +11,15 @@ def spaceapi(bot, target=None):
 
     with aiohttp.Timeout(10):
         with aiohttp.ClientSession(loop=bot.loop) as session:
-            resp = yield from session.get(config['url'])
-            if resp.status != 200:
-                if target is not None:
-                    bot.privmsg(target, "Error while retrieving spaceapi data")
-                raise Exception()
-            r = yield from resp.read()
-    return json.loads(r.decode('utf-8'))
+            ret = []
+            for space_url in config['url'].split():
+                msg = "Error while retrieving spaceapi data" + space_url
+                resp = yield from session.get(space_url)
+                if resp.status != 200:
+                    if target is not None:
+                        bot.privmsg(target, msg)
+                    raise Exception()
```

Comment:
This ideally should throw a custom exception, not a generic one

Reply:
Well it is a good idea. You can add this.